src/main/scala/cromwell/engine/CallActor.scala

```diff
@@ -42,7 +42,7 @@ class CallActor(key: CallKey, locallyQualifiedInputs: CallInputs, backend: Backe
 
   val call = key.scope
   val callReference = CallReference(workflowDescriptor.name, workflowDescriptor.id, call.fullyQualifiedName)
-  val tag = s"CallActor [$callReference]"
+  val tag = s"CallActor [$callReference${key.index.map(i => s" (shard $i)").getOrElse("")}]"
```

Comment:
I actually didn't mean to leave this in... I removed it in a pre-PR fixup but you caught it already!